SR2025/2025-01-15-check-in.md

```diff
@@ -0,0 +1,27 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 {TLA} January Check-in
```

Comment:
Suggestion: Much of this email is actually reminding teams. The subject should reflect that.

Reply:
Thanks, however the subject of this email reflects the focus of the email. This is a 1:1 personalised check-in email aiming to get a view on the status of teams, much like the ones earlier in the year.

The reminders are secondary (even if they form a large part of the text).